### PR TITLE
fix: add Shivani1917

### LINKED ISSUE #1500: Shivani Badola
githubprofile-shivanibadola

### PR DESCRIPTION
**Pull request by Shivani1917**
githubprofile-shivanibadola

resolves #1500

### CODE REVIEW

File: profiles/Shivani1917.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/79411680?s=400&u=44fbd7b8021b67164aed1f19b30db72071400c7c&v=4",
+    "issueId": 1500,
+    "name": "Shivani Badola",
+    "username": "Shivani1917"
+  }
```
Comment (Panquesito7) [inline profiles/Shivani1917.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: profiles/Shivani1917.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://avatars.githubusercontent.com/u/79411680?s=400&u=44fbd7b8021b67164aed1f19b30db72071400c7c&v=4",
+    "issueId": 1500,
+    "name": "Shivani Badola",
+    "username": "Shivani1917"
```
Comment (Panquesito7) [inline profiles/Shivani1917.json:5]: It should be a two-space indentation and not four.
```suggestion
  "image": "https://avatars.githubusercontent.com/u/79411680?s=400&u=44fbd7b8021b67164aed1f19b30db72071400c7c&v=4",
  "issueId": 1500,
  "name": "Shivani Badola",
  "username": "Shivani1917"
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**